guake/utils.py

```diff
@@ -20,12 +20,15 @@
 Boston, MA 02110-1301 USA
 """
 import datetime
+import enum
 import logging
 import os
 import platform
 import subprocess
 import time
 
+import cairo
```

Comment:
Does this need to be added to requirements.txt? Looks like a new dependency.

Reply:
requirements.txt is maintained by pipenv

Reply:
Oh alright, just @la5942's review stands then.